examples/vue/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "devDependencies": {
     "@vitejs/plugin-vue": "^1.2.5",
+    "@types/node": "18.8.0",
```

Comment:
Just curious, why was this added?

Reply:
https://github.com/johnsoncodehk/volar/issues/464

Reply:
Sounds good, let's keep an eye on this issue and remove this after they fix it.

Reply:
Sounds like this dependency is already being installed by vue, is that correct? Could we get around this by putting it in `resolutions` instead of taking a `devDependency` on it? That feels more appropriate for a workaround like this, since we won't remember if we need this or not in the future.